doc/installation.rst

```diff
@@ -12,18 +12,30 @@ Linux
 Installation with `pip3`
 ========================
 
-This is the recommended installation method using the 'package installer for Python' `pip3`:
-
-::
-
-  pip3 install --user eoshep
-
-You can now use the EOS Python module via ``import eos``. In addition, some command-line scripts have become available.
-
-| To **update** EOS, add the flag ``--upgrade`` to the above command.
-| To install the **development version**, add the flag ``--pre``.
-  This version is automatically up-to-date with the master branch in the Github EOS repository.
-| You can **test** and debug the installation of the python module as described :ref:`here <faq-check-installation>`.
+This is the recommended installation method using the 'package installer for Python' `pip3`.
+We distribute a release version and a developer version.
```

Comment:
It should say *development version* instead, since the distributed versions are not for developers but for users.

Reply:
I agree